cupy_setup_build.py

```diff
@@ -654,64 +654,44 @@ def get_long_description():
 def prepare_wheel_libs():
     """Prepare shared libraries and include files for wheels.
 
-    On Windows, DLLs will be placed under `cupy/cuda`.
-    On other platforms, shared libraries are placed under `cupy/.data/lib` and
-    RUNPATH will be set to this directory later.
+    Shared libraries are placed under `cupy/.data/lib` and
+    RUNPATH will be set to this directory later (Linux only).
```

Comment:
Windows is the same path?

Reply:
Here I meant `RUNPATH will be set to this directory` is not applicable for Windows.